src/backend/aspen/workflows/nextstrain_run/builds_templates/mega_template.yaml

```diff
@@ -35,6 +35,8 @@ refine:
   
 priorities:
   crowding_penalty: {crowding_penalty_per_tree_type}
+  
+include_hcov19_prefix: True
```

Comment:
the way this works is it builds the tree with all original sample names, and in the very end add `hCoV-19` prefix to all sample names unless it already has it.

Reply:
we'll let CZGE add this instead of Nextstrain